cli/src/generate/templates/build.rs

```diff
@@ -21,20 +17,4 @@ fn main() {
 
     c_config.compile("parser");
```

Comment:
```suggestion
    c_config.compile("PARSER_NAME");
```

Reply:
maybe tree-sitter-PARSER_NAME?

Reply:
It doesn't matter much since this just generates a static file.